atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -265,28 +264,25 @@ public String toString() {
 
         poolConfig.setJmxNamePrefix(CassandraLogHelper.host(host));
         GenericObjectPool<CassandraClient> pool = new GenericObjectPool<>(cassandraClientFactory, poolConfig);
-        registerMetrics(pool, poolNumber);
+        registerMetrics(pool, host.getHostString());
```

Comment:
shouldn't be adding hosts as a tag. talked with @fsamuel-bs about this

Reply:
For SA on the PR, spoke to Samuel offline, we dont want to have host names as tags as that will blow up the metricName*tagName cardinality which is expensive for the internal metrics framework. We can continue using poolNumbers as tags but that will cause confusion if a pool is removed and another added, this will switch around the metric names, so if pool1 is removed and pool4 added, we will see no more metrics for pool4, but pool4 will try to  report 
to pool3 and cause a lot of confusion.

We should fix this up or rely on logging to get us the information here.